torch/nn/modules/instancenorm.py

```diff
@@ -114,8 +127,8 @@ class InstanceNorm1d(_InstanceNorm):
             statistics in both training and eval modes. Default: ``False``
 
     Shape:
-        - Input: :math:`(N, C, L)`
-        - Output: :math:`(N, C, L)` (same shape as input)
+        - Input: :math:`(N, C, L)` or :math:`(C, L)`
```

Comment:
`num_features` doc needs to be updated above. It falsely claims that an input of shape `(N, L)` is possible. Currently, an error is thrown for a 2D input, and after this PR, a 2D input will be treated as `(C, L)`.

Same change is needed for the `LazyInstanceNorm1d` docs.